src/mailroom.py

```diff
@@ -42,9 +44,31 @@ def send_thanks():  # pragma: no cover
             else:
                 donor_dict.setdefault(donor, []).append(amount)
             print('')
-            print(generate_thankyou(donor, amount))
-            print('-Sincerely, Gov. Whoever')
-            press_to_continue()
+            print(generate_thankyou(donor, format_amount(amount)))
+
+
+def generate_thankyou(donor, amount, template='letter_template.txt'):
+    """Print the template_letter.txt file or a default thank you."""
+    file_found = True
```

Comment:
If I am not mistaken, from line 53 to 58 the script will try to read file `letter_template.txt` at the `mailroom` directory, and if there isn't one, the script will look for that file in the `src` folder correct?

There are 2 problems if this is the case:

1. It would be confusing if there's a `letter_template.txt` at both root directory and `src` folder, so I think we should only check if the template exists at the `mailroom/src` folder.
2. If we execute `mailroom.py` from somewhere other than the `mailroom` and `mailroom/src` folders, the script won't be able to find the `letter_template.txt`. For example: execute `python mailroom/src/mailroom.py` from 1 directory above `mailroom` will default to the `DEFAULT_LETTER` template

**Propose changes:**
```python
def generate_thankyou(donor, amount, template='letter_template.txt'):
    try:
        f = open('{}/{}'.format(
            os.path.dirname(os.path.abspath(__file__)), template), 'r'
        )
        letter = ''
        for line in f:
            if line[0] != '#':
                letter += line
        f.close()
    except IOError:
        letter = DEFAULT_LETTER
    return letter.format(name=donor, amount=amount)
```
For this code, the script will only check if there's file named `letter_template.txt` in the same directory with `mailroom.py` file